test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -101,9 +104,15 @@ def test_network_tokenization_transcript_scrubbing
 
   def test_successful_authorization
     assert response = @gateway.authorize(@amount, @credit_card, @options)
-    assert_equal 'Successful transaction', response.message
-    assert_success response
-    assert response.test?
+    assert_successful_response(response)
+    assert !response.authorization.blank?
```

Comment:
Consider changing this to `assert_not` instead of `assert !`.

Reply:
That's actually not code that I added, so I'll leave it alone for now since ```assert !``` is used consistently (15 times) throughout the tests.

Reply:
(I'm slightly bewildered as to why GitHub shows this with a green highlight, but it is indeed in master here: https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_cyber_source_test.rb#L107).

Reply:
Is this something worth refactoring for readability?

Reply:
I do like `assert_not` better in general too, but I think consistency within the same test case is more important, so I would stick with `assert !` if that's what's used throughout the rest of the nearby code.

Reply:
Yes, agreed that consistency is most important! My question was whether this is something that would be worth changing throughout this test file.

Reply:
Ah, gotcha. Doesn't hurt either way in my opinion, but probably in this case it is a matter of deciding how far to go for the immediate ticket vs. leaving otherwise unrelated changes to a different refactor.

I know I always see more I want to do, but often have to determine what is necessary for the task at hand and constrain myself to that. 🙂